website/docs/r/proximity_placement_group.html.markdown

```diff
@@ -40,6 +40,12 @@ The following arguments are supported:
 
 * `location` - (Required) Specifies the supported Azure location where the resource exists. Changing this forces a new resource to be created.
 
+* `intent_vm_sizes` - (Optional) Specifies the supported sizes of Virtual Machines that can be created in the Proximity Placement Group.
```

Comment:
I'm not sure intent is very clear here? would this make more sense as
```suggestion
* `allowed_vm_sizes` - (Optional) Specifies the supported sizes of Virtual Machines that can be created in the Proximity Placement Group.
```

Reply:
updated